Algorithm.CSharp/MarginCallEventsAlgorithm.cs

```diff
@@ -77,7 +77,7 @@ public override void OnMarginCall(List<SubmitOrderRequest> requests)
         }
 
         /// <summary>
-        /// Margin call warning event handler. This method is called when Portoflio.MarginRemaining is under 5% of your Portfolio.TotalPortfolioValue
+        /// Margin call warning event handler. This method is called when Portfolio.MarginRemaining is under 5% of your Portfolio.TotalPortfolioValue
```

Comment:
hehe -- I bet this traces back to me, for some reason I was always bad at typing portfolio :p